autoload/neomake.vim

```diff
@@ -645,7 +648,22 @@ function! s:AddExprCallback(jobinfo, prev_index) abort
         let index += 1
 
         let before = copy(entry)
-        call call(s:postprocess, [entry], maker)
+        if type(s:postprocess) == type([])
+            let s = s:postprocess
+            for s:f in s:postprocess
+                if type(s:f) == type({})
+                    let s:this = extend(copy(s:f), {'maker': maker})
+                    call call(s:f.fn, [entry], s:this)
+                else
+                    let s:this = {'maker': maker}
+                    call call(s:f, [entry], s:this)
+                endif
+            endfor
+            " TODO: abort on first invalid entry already?!
```

Comment:
If one postprocess sets valid=--1 we might abort here already.
Bot OTOH another one might revert it?!